### PR TITLE
Enhancement style

### PR DESCRIPTION
**Pull request by alsturm**

### CODE REVIEW

**Review by taulinger — COMMENTED** [review]
Looks good, just one test is missing ...

File: src/modules/baseLayer/components/switcher/BaseLayerSwitcher.js [hunk 1]
```diff
@@ -95,8 +100,8 @@ export class BaseLayerSwitcher extends MvuElement {
 
 			return html`
 				<style>${css}</style>
-				<div class="title">
-					Basiskarte
+				<div class="title">					
+					${translate('baselayer_switcher_header')}
```
Comment (taulinger) [inline src/modules/baseLayer/components/switcher/BaseLayerSwitcher.js:104]: no test for this key (baselayer_switcher_header)

File: src/modules/header/i18n/header.provider.js [hunk 2]
```diff
@@ -6,25 +6,27 @@ export const provide = (lang) => {
 				//the first part of the snake_case key should be the name of the related module
 				header_tab_topics_button: 'Topics',
 				header_tab_topics_title: 'Open topics',
-				header_tab_maps_button: 'Layers configuration',
-				header_tab_maps_title: 'Open layers configuration',
+				header_tab_maps_button: 'Map',
+				header_tab_maps_title: 'Open map',
```
Comment (taulinger) [inline src/modules/header/i18n/header.provider.js:10]: Would suggest "Open map configuration"

File: src/modules/header/i18n/header.provider.js [hunk 3]
```diff
@@ -6,25 +6,27 @@ export const provide = (lang) => {
...
 
 		case 'de':
 			return {
 				//the first part of the snake_case key should be the name of the related module
 				header_tab_topics_button: 'Themen',
 				header_tab_topics_title: 'Themen öffnen',
-				header_tab_maps_button: 'Ebenenverwaltung',
-				header_tab_maps_title: 'Ebenenverwaltung öffnen',
+				header_tab_maps_button: 'Karte',
+				header_tab_maps_title: 'Karte öffnen',
```
Comment (taulinger) [inline src/modules/header/i18n/header.provider.js:24]: Would suggest "Kartenverwaltung öffnen"

File: src/modules/header/components/Header.js [hunk 4]
```diff
@@ -195,7 +195,7 @@ export class Header extends MvuElement {
 					<div class="header__background">
 					</div>
 					<div class='header__search-container'>
-						<input id='input' @focus="${onInputFocus}" @blur="${onInputBlur}" @input="${onInput}" class='header__search' type="search" placeholder="" />          
+						<input id='input' placeholder='${translate('header_search_placeholder')}' @focus="${onInputFocus}" @blur="${onInputBlur}" @input="${onInput}" class='header__search' type="search" placeholder="" />
```
Comment (taulinger) [inline src/modules/header/components/Header.js:198]: no test for this key (header_search_placeholder)

**Review by taulinger — COMMENTED** [review]
Very nice proceedings!

File: src/modules/header/components/Header.js (hunk 4 above)
Comment (alsturm) [reply src/modules/header/components/Header.js:198]: there is one in header.provider.test?

Comment (taulinger) [reply src/modules/header/components/Header.js:198]: Meant in `Header.test.js` -> input should contain  placeholder attribute containing the correct i18n value

**Pull request merged**